van-gogh-gallery/josefin/pages/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>about</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-container">
+                <li><a href="/van-gogh-gallery/josefin/index.html" class="nav-link">Home</a></li>
+                <li><a href="#" class="nav-link">Tickets & Visit</a></li>
+                <li><a href="#" class="nav-link">Art & Stories</a></li>
+                <li><a href="pages/about.html" class="nav-link">About</a></li>
```

Comment:
We're alredy on about 
```suggestion
                <li><a href="#" class="nav-link">About</a></li>
```